tests/sanity_helpers.py

```diff
@@ -1,133 +1,73 @@
 import logging
 
+from ocs_ci.ocs.ocp import OCP
 from ocs_ci.framework import config
 from ocs_ci.ocs import constants, node
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
 from ocs_ci.utility.utils import ceph_health_check
 from ocs_ci.ocs.cluster import CephCluster
 
-from tests import helpers
-
 
 logger = logging.getLogger(__name__)
 
 
-def health_check(nodes):
+class Sanity:
```

Comment:
inherit from?

Reply:
This is not a test class, then should inherit 'Object'